<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileListClass</class>
 <widget class="QWidget" name="fileListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>fileList</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton { /* all types of tool button */ 
color: #050505;
border: 0px solid #2e3236; /*#848584;*/
border-radius: 0px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #e7e7e7, stop:0.45 #c8c9c9, stop:0.5 #bcbebc, stop: 1 #818382);
min-height: 18px;
min-width: 30px;
}

QPushButton:hover { /* all types of tool button */
color: #050505;
border: 1px solid #a75e16; /*#848584;*/
border-radius: 0px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #e7e7e7, stop:0.45 #c8c9c9, stop:0.5 #bcbebc, stop: 1 #818382);
}

/*QPushButton:pressed {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #737372, stop:0.45 #b0b1b0, stop:0.5 #c2c1c2, stop: 1 #c2c1c2);
color: #050505;
border: 1px solid #2e3236;
}

QPushButton:checked {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #737372, stop:0.45 #b0b1b0, stop:0.5 #c2c1c2, stop: 1 #c2c1c2);
color: #050505;
border: 1px solid #2e3236;
}*/

QPushButton:pressed {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #f4d4b4, stop:0.45 #b77738, stop:0.5 #a7601b, stop: 1 #884d12);
/*background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #267194, stop:0.45 #307FA4, stop:0.5 #3987AC, stop: 1 #549EC1); */
color: WHITE;
border: 0px solid #124159;
}

QPushButton::checked {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #f4d4b4, stop:0.45 #b77738, stop:0.5 #a7601b, stop: 1 #884d12);
color: WHITE;
border: 0px solid #124159;
}

QLabel 
{ 
color: rgb(215, 215, 215); 
} 

QTimeEdit {
        background: transparent;
        border: 0px solid #383B3D;
        color: rgb(215, 215, 215); 
}

QTableWidget { 
show-decoration-selected: 1; /* make the selection span the entire width of the view */ 
} 
QTableWidget::item:alternate { 
background: #EEEEEE; 
} 
QTableWidget::item:selected { 
border: 1px solid #363636; 
} 
QTableWidget::item:selected:!active { 
background: rgb(54, 54, 54); 
} 
/* Активный итем*/ 
QTableWidget::item:selected:active { 
background: rgb(54, 54, 54); 
} 

/*QTableWidget::item:hover 
{ 
background: rgb(75, 45, 32); 
} */
/* Наведение на активный итем*/ 
/*QTableWidget::item:hover:selected:active{ 
background: rgb(75, 45, 32); 
} */
QScrollBar:vertical { 
border: 1px solid black; 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(60, 60, 60, 255), stop:1 rgba(50, 50, 50, 255)); 
width: 13px; 
margin: 12px 3px 12px 3px; 
} 
QScrollBar::handle:vertical { 
border: 1px solid black; 
border-radius: 3px; 
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(194, 194, 194, 255), stop:0.517045 rgba(221, 221, 221, 255), stop:1 rgba(194, 194, 194, 255));
/*background: transparent;*//*qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(198, 198, 198, 255), stop:1 rgba(193, 193, 193, 255)); */
margin: 0 -3px; 
min-height: 10px;
max-height: 10px;
 image: url(:/images/scroll_handle_2.png);
/* margin: 0 22px 0 22px;*/ 
} 

QScrollBar::add-line:vertical 
{
      /*border: 2px solid grey;*/
      background: transparent;
      height: 11px;
	  width: 11px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
	 image: url(:/images/scroll_down.png);
  }

 QScrollBar::sub-line:vertical 
{
      background: transparent;
      height: 11px;
	  width: 11px;
      subcontrol-position: top;
    subcontrol-origin: margin;
	 image: url(:/images/scroll_up.png);
  }

/*  QScrollBar::add-line:vertical {
     
      background: transparent;
      height: 11px;
	  width: 11px;
      subcontrol-position: bottom;
    subcontrol-origin: margin;
	 image: url(:/images/scroll_down.png);
  }*/

QScrollBar::up-arrow:vertical
{
      width: 11px;
      height: 11px;
  }

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }

/*QScrollBar::down-arrow:vertical {
      width: 11px;
      height: 11px;
 	  image: url(:/images/scroll_down.png);
  }*/

QCheckBox {
     spacing: 5px;
 }

 QCheckBox::indicator {
     width: 13px;
     height: 13px;
 }

 QCheckBox::indicator:unchecked {
     image: url(:/images/chbNotChecked.png);
 }

 QCheckBox::indicator:unchecked:hover {
     image: url(:/images/chbNotChecked.png);
 }

 QCheckBox::indicator:unchecked:pressed {
     image: url(:/images/chbChecked.png);
 }

 QCheckBox::indicator:checked {
     image: url(:/images/chbChecked.png);
 }

 QCheckBox::indicator:checked:hover {
     image: url(:/images/chbChecked.png);
 }

 QCheckBox::indicator:checked:pressed {
     image: url(:/images/chbChecked.png);
 }

 QCheckBox::indicator:indeterminate:hover {
     image: url(:/images/chbChecked.png);
 }

 QCheckBox::indicator:indeterminate:pressed {
     image: url(:/images/chbChecked.png);
 }

QSlider::groove:horizontal {
	
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.488636 rgba(128, 128, 128, 255), stop:1 rgba(123, 123, 123, 255));
position: absolute;
height: 4px;
left: 2px; right: 2px;
border: 1px solid black;
margin: 0px 0;
}
QSlider::handle:horizontal {
/*height: 6px;
width: 10px;
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 170, 255, 255), stop:0.19397 rgba(0, 85, 127, 255), stop:0.504819 rgba(0, 170, 255, 255), stop:0.79 rgba(0, 213, 255, 255), stop:1 rgba(255, 255, 255, 255));
border: 1px solid #686868;
border-radius: 3px;
margin: -1px 0; 
border-radius: 3px;*/
image: url(:/images/slider.png);
margin: -3px 0;
}

 QSlider::add-page:horizontal {
     background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.488636 rgba(128, 128, 128, 255), stop:1 rgba(123, 123, 123, 255));
border: 1px solid black;
 }

 QSlider::sub-page:horizontal {
     /*background:  rgb(254, 191, 0);*/
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #f4d4b4, stop:0.45 #b77738, stop:0.5 #a7601b, stop: 1 #884d12);
	 border: 1px solid black;
 }

  </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbRec">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Record IQ Wave file</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Rec</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPlay">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Play IQ Wave file</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPause">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Pause</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Stop</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbBack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Rewind</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbLoop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Loop</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbAddFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDelFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Del</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>4</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget { 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(33, 33, 33, 255), stop:0.5 rgba(65, 65, 65, 255), stop:0.943182 rgba(33, 33, 33, 255));
border: 0px solid #124159;
color: #E5E5E5;
}
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QSlider" name="slIqTrack">
       <property name="value">
        <number>40</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="teIqTime">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>4</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget { 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(33, 33, 33, 255), stop:0.5 rgba(65, 65, 65, 255), stop:0.943182 rgba(33, 33, 33, 255));
border: 0px solid #124159;
color: #E5E5E5;
}
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="twList">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
